static/js/containers/App_test.js

```diff
@@ -38,29 +41,37 @@ import IntegrationTestHelper from '../util/integration_test_helper';
 
 describe('App', () => {
   let listenForActions, renderComponent, helper;
-  let editProfileActions;
+  let editProfileActions, redirectActions;
 
   beforeEach(() => {
     helper = new IntegrationTestHelper();
     listenForActions = helper.listenForActions.bind(helper);
     renderComponent = helper.renderComponent.bind(helper);
-    editProfileActions = [
+
+    const successActions = [
+      REQUEST_GET_PROGRAM_ENROLLMENTS,
+      RECEIVE_GET_PROGRAM_ENROLLMENTS_SUCCESS,
+      REQUEST_GET_USER_PROFILE,
+      RECEIVE_GET_USER_PROFILE_SUCCESS,
+    ];
+    editProfileActions = successActions.concat([
       START_PROFILE_EDIT,
       START_PROFILE_EDIT,
       UPDATE_PROFILE_VALIDATION,
       SET_PROFILE_STEP,
-    ];
+    ]);
+    redirectActions = successActions.concat([
+      START_PROFILE_EDIT,
+    ]);
```

Comment:
looks like you're assigning expected actions in the `describe` handler in some test files, and inside the `beforeEach` handler in others. it doesn't look like you're actually changing these lists anywhere - when you need to add to the list, you are setting a different variable, which i think is the right approach. can we make this consistent?